nextflow.config

```diff
@@ -231,13 +231,13 @@ dag {
 }
 
 manifest {
-    name            = 'nf-core/staramr'
-    author          = """Eric Marinier"""
-    homePage        = 'https://github.com/nf-core/staramr'
+    name            = 'phac-nml/staramrnf'
+    author          = """Steven Sutcliffe"""
+    homePage        = 'https://https://github.com/phac-nml/staramrnf'
```

Comment:
You have an extra `https://` prefix here.

Reply:
b41700c